pom.xml

```diff
@@ -208,10 +208,10 @@
       <version>v1-rev${container.revision}-${google.api.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.yaml</groupId>
-      <artifactId>snakeyaml</artifactId>
-      <version>1.19</version>
-    </dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>snakeyaml-api</artifactId>
+      <version>1.27.0</version>
+      <type>hpi</type>
```

Comment:
```suggestion
```

No need for it, otherwise you have to update the doc: https://plugins.jenkins.io/snakeyaml-api/ ;)